### DISCUSSION TITLE
Hyprspace under systemd control

### DISCUSSION THREAD

**Discussion by benjiworld** [alecbcs/hyprspace#77] [Ideas]
Hello everybody.. Hyprspace is really a great tool to connect our nodes! Any way to create a service via systemd to make it to survive on reboots? Or any other solution will be welcome! Thank you in advance.

**Comment by alecbcs** [comment]
Hi @benjiworld! I believe the systemd service under [examples/systemd](https://github.com/hyprspace/hyprspace/blob/main/examples/systemd) is what you're looking for. Copy that file to

```
/lib/systemd/system/hyprspace@.service
```

 and then after creating a hyprspace interface (`sudo hyprspace init hs0`) you can have that interface automatically start up by running,

```
sudo systemctl enable --now hyprspace@YOUR_INTERFACE_NAME
```
